source/includes/api/_widget.md

```diff
@@ -582,281 +595,543 @@ curl -X GET https://api.resourcewatch.org/v1/widget?includes=metadata
             "published": true,
             "freeze": false,
             "env": "production",
-            "queryUrl": null,
-            "widgetConfig": "{}",
+            "queryUrl": "/v1/query?sql=Select * from data",
+            "widgetConfig": {
+                "data": {
+                    "url": "/v1/query?sql=Select * from data"
+                }
+            },
             "template": false,
             "layerId": null,
             "createdAt": "2017-02-08T15:30:34.505Z",
-            "updatedAt": "2017-02-08T15:30:34.505Z",
-            "metadata": [
-              {
-                "id": "5aeb1c74a096b50010f3843f",
-                "type": "metadata",
-                "attributes": {
-                  "dataset": "86777822-d995-49cd-b9c3-d4ea4f82c0a3",
-                  "application": "rw",
-                  "resource": {
-                    "id": "51851e22-1eda-4bf5-bbcc-cde3f9a3a943",
-                    "type": "widget"
-                  },
-                  "language": "en",
-                  "info": {
-                    "caption": "t",
-                    "widgetLinks": []
-                  },
-                  "createdAt": "2018-05-03T14:28:04.482Z",
-                  "updatedAt": "2018-06-07T11:30:40.054Z",
-                  "status": "published"
-                }
-              }
-            ]
-          }
-        }]
+            "updatedAt": "2017-02-08T15:30:34.505Z"
+        }
     }
 }
 ```
 
-#### Requesting multiple additional entities
+When getting a single widget, you can optionally provide a `queryUrl` query parameter. If provided, this parameter will overwrite the following response values:
 
-You can request multiple related entities in a single request using commas to separate multiple keywords
+- `data.attributes.queryUrl`
+- `data.attributes.widgetConfig.data.url`
+- `data.attributes.widgetConfig.data[0].url`
+ 
+This overwrite will only happen if a previous value already existed for the respective field. Also, keep in mind that these changes will NOT be persisted to the database, and will only affect the current request's response. 
+ 
+### Customize the query url of a widget response
```

Comment:
Huh..what's the purpose of this (and the above overwrite functionality)?

Reply:
@aagm tells me that this was requested to make widgets more customizeable - instead of having different widgets for variations of the same visualization, this allows tweaking the same widget without duplicating it. As to why this logic is in the API as opposed to the client apps using it - the best reason we could come up with was to avoid having to duplicate this logic on potentially multiple clients.